nbsite/tmplate2/conf.py

```diff
@@ -8,19 +8,21 @@
 
 project = u'Project'
 authors = u'Authors for html meta author'
-copyright = u'2017 ' + authors
+copyright = u'2019 ' + authors
 description = 'Short description for html meta description.'
-
-version = '0.0.1'
-release = '0.0.1'
+version = release = '0.0.1'
 
 html_static_path += ['_static']
-html_theme = 'sphinx_ioam_theme'
+html_theme = 'sphinx_pyviz_theme'
```

Comment:
Is tmplate2 specific to pyviz?  I'd like the default template in nbsite not be the pyviz_theme, so that non-pyviz users of nbsite (when there are any) don't create sites that appear to be pyviz.  There should at least be *some* visual differences between the default and the way pyviz sites look.

Reply:
Yeah I've been wondering about this. Tmplate2 is used when initing with the new style. I think it is very helpful for us to see what the options for the theme are and what not, but maybe the pyviz theme should require a flag. Like `nbsite init --theme pyviz`.

Reply:
That's what I had in mind, thanks.